### PR TITLE
[Internal] Improved defensive checks in ActivityDigestWorker queueworker.

### PR DESCRIPTION
**Pull request by navneet0693**
## Problem
Have more defensive checks in ActivityDigestWorker.

## Solution
Improve defensive checks.

## Issue tracker
N.A

## How to test
- [x] Queues should work normally.

## Screenshots
N.A

## Release notes
N.A

## Change Record
N.A

## Translations
N.A

### CODE REVIEW

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2641-tt85jm6wbhpbvld5cwrydo0la06vrltd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b087c5821ec85cda0bd494

File: modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivityDigestWorker.php [hunk 1]
```diff
@@ -127,7 +134,7 @@ public function processItem($data) {
           if (
           ($activity instanceof Activity) &&
             ($activity->getRelatedEntity() !== NULL) &&
-            !$activity->getRelatedEntity()->access('view', $target)
+          ($activity->getRelatedEntity()->access('view', $target) === FALSE)
```
Comment (bojanbogdanovic) [inline modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivityDigestWorker.php:137]: @navneet0693 could you fix plz the inconsistent indentation on these statements?

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2641-tt85jm6wbhpbvld5cwrydo0la06vrltd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b087c5821ec85cda0bd494

File: modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivityDigestWorker.php [hunk 2]
```diff
@@ -126,9 +133,8 @@ public function processItem($data) {
           // Only for users that have access to related content.
           if (
           ($activity instanceof Activity) &&
-            ($activity->getRelatedEntity() !== NULL) &&
-            !$activity->getRelatedEntity()->access('view', $target)
-          ) {
+          ($activity->getRelatedEntity() !== NULL) &&
+          ($activity->getRelatedEntity()->access('view', $target) === FALSE)) {
```
Comment (Ressinel) [inline modules/custom/activity_send/modules/activity_send_email/src/Plugin/QueueWorker/ActivityDigestWorker.php:137]: @navneet0693  Let's do the same code style as you added above:
```
if (($activity instanceof Activity) &&
  ($activity->getRelatedEntity() !== NULL) &&
  ($activity->getRelatedEntity()->access('view', $target) === FALSE)
) {
...
```
or
```
if (
  ($activity instanceof Activity) &&
  ($activity->getRelatedEntity() !== NULL) &&
  ($activity->getRelatedEntity()->access('view', $target) === FALSE)
) {
...
```

**Review by Ressinel — CHANGES_REQUESTED** [review]

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2641-tt85jm6wbhpbvld5cwrydo0la06vrltd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b087c5821ec85cda0bd494

**Comment by open-social-tugboat** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2641-tt85jm6wbhpbvld5cwrydo0la06vrltd.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61b087c5821ec85cda0bd494

**Review by Ressinel — APPROVED** [review]
Looks good to me.

**Comment by navneet0693** [comment]
🍒 -picked to 10.3.x, 10.2.x and 10.1.x

**Comment by ronaldtebrake** [comment]
Don't roget 11.0.x ;)

**Comment by navneet0693** [comment]
@ronaldtebrake Thanks for reminding me.

**Pull request merged**